hazelcast/src/hazelcast/client/internal/socket/SSLSocket.cpp

```diff
@@ -130,8 +120,10 @@ namespace hazelcast {
                         // though the connect operation notionally succeeded. Therefore we must
                         // check whether the socket is still open before deciding if we succeeded
                         // or failed.
-                        if (ec) {
-                            return ec.value();
+                        if (ioRunErrorCode)
```

Comment:
Is this code style intentional? It could not follow the logic really, but it looks like a leftover.

Reply:
Yes, that was left over. I corrected it now.